### PR TITLE
BP-2507 Add Payment initiation

### PR DESCRIPTION
**Pull request by Rinor12010**

### CODE REVIEW

File: src/PaymentMethods/PaymentInitiation/PaymentInitiation.php [hunk 1]
```diff
@@ -0,0 +1,24 @@
+<?php
+
+namespace Buckaroo\PaymentMethods\PaymentInitiation;
+
+use Buckaroo\Models\Model;
+use Buckaroo\PaymentMethods\PaymentInitiation\Models\Pay;
+use \Buckaroo\PaymentMethods\PayablePaymentMethod;
```
Comment (ShuCh3n) [inline src/PaymentMethods/PaymentInitiation/PaymentInitiation.php:7]: Please remove slash

File: src/PaymentMethods/PaymentMethodFactory.php [hunk 2]
```diff
@@ -97,12 +98,12 @@ class PaymentMethodFactory
         KBC::class => ['kbcpaymentbutton'],
         Paypal::class => ['paypal'],
         PayPerEmail::class => ['payperemail'],
+        PaymentInitiation::class => ['paymentinitiation'],
         EPS::class => ['eps'],
         Emandates::class => ['emandates'],
         Sofort::class => ['sofort', 'sofortueberweisung'],
         Tinka::class => ['tinka'],
         Marketplaces::class => ['marketplaces'],
-        NoServiceSpecifiedPayment::class => ['noservice'],
```
Comment (ShuCh3n) [inline src/PaymentMethods/PaymentMethodFactory.php:105]: Why is this line removed?

**Review by ShuCh3n — CHANGES_REQUESTED** [review]
Please check the comments

File: src/PaymentMethods/PaymentMethodFactory.php (hunk 2 above)
Comment (Rinor12010) [reply src/PaymentMethods/PaymentMethodFactory.php:105]: It's not necessary if you want to make a payment with no method specified just don't send a method name there is no need to check for the noservice method

Comment (ShuCh3n) [reply src/PaymentMethods/PaymentMethodFactory.php:105]: Yes, but it's more readable for some programmers passing a name to the method.

**Review by ShuCh3n — APPROVED** [review]
